circle.yml

```diff
@@ -0,0 +1,9 @@
+machine:
+  ruby:
+    version: 2.3.3
```

Comment:
What do you think about using ruby 2.3.4? it's the latest 2.3 release and ideally we'll want to use it over 2.3.3. 

Or, we could be super hip and trendy and use 2.4.1.

https://circleci.com/docs/1.0/build-image-trusty/#ruby All of those versions are preinstalled so build times on CircleCI will be low :)